nautobot_device_lifecycle_mgmt/models.py

```diff
@@ -345,7 +403,14 @@ def expired(self):
         """Return True or False if chosen field is expired."""
         if not self.end:
             return False
-        return datetime.today().date() >= self.end
+        return datetime.today().date() > self.end
+
+    @property
+    def active(self):
+        """Return True or False if chosen field is active."""
```

Comment:
Could you be more explicit here as to what this property does?

Reply:
Yes, this comment is a bit lacking. I've updated with more details.